categories/models.py

```diff
@@ -10,10 +10,24 @@ class Category(TranslatableModel):
         description=models.TextField(verbose_name=_("description"), blank=True),
     )
 
+    principle = models.BooleanField(default=False)
+
     class Meta:
         verbose_name = _("category")
         verbose_name_plural = _("categories")
-        ordering = ("id",)
+        ordering = (
+            "-principle",
+            "translations__name",
```

Comment:
can't find any proof right now, but I think it is not reliable to order by translated fields. you might not have translations for all objects in certain language and then the default translation fallback might mess up the result or just result in an error... If you are 100% sure, that this is safe though, we could keep it